documents/data-sharing-annual-community-survey.md

```diff
@@ -0,0 +1,70 @@
+# Annual community survey data sharing
+
+This document describes how data from the annual survey should be shared.
+
+## Objectives
+
+In priority order:
+
+* Protect the privacy of respondents.
+* Provide high quality data (unbiased data which is unlikely to be misinterpreted, c.f., raw data).
+* Provide useful and actionable data to the Rust project (i.e., teams and working groups, contributors, and the Foundation).
+* Provide data in a timely manner.
+* Provide data which is generally useful for the community (users and potential users of Rust, and providers of Rust tools and services). Note that this does not mean producing data to be useful for specific segments of the community.
+* Provide marketing for Rust.
+* Avoid overloading people working directly with the survey data (note that in 2021 there were over 9000 completed surveys and multiple opportunities for free form responses).
+
+Non-goals:
+
+* Making money by selling the data in any form.
+* The project will aim to not privilege companies which employ people with access to survey data (raw or processed). However, we acknowledge that controlling widely shared data is impossible.
+* Directly supporting commercial products or companies with data.
+* Produce a rigorous or complete statistical study of survey results.
```

Comment:
I still think we're missing a non-goal like:
* provide individuals or specific groups with data that is likely only beneficial to them and not to the wider community 

We don't want to be in the business of servicing ad-hoc requests with the exception of ad-hoc requests for project teams (and even then we'll want to set limits).